src/config/dynamic-theme-fixes.config

```diff
@@ -1674,6 +1674,9 @@ code:first-of-type {
 .ProseMirror hr.ak-editor-selected-node {
     background-color: rgb(0, 101, 255) !important;
 }
+.ghx-flagged {
+    background-color: rgb(0, 0, 0) !important;
```

Comment:
Hmm actually, I think it's more interesting to fix the issue of why it's becoming red instead of darker yellow. As making it now black by-default isn't correct either.